1.9/deploying-services/pods/technical-overview.md

```diff
@@ -17,7 +17,7 @@ You configure a pod via a pod definition, which is similar to a Marathon applica
 # Networking
 Marathon pods only support the [DC/OS Universal container runtime](/docs/1.9/deploying-services/containerizers/), which supports multiple image formats, including Docker.
 
-The Universal container runtime simplifies networking by allowing the containers of each pod instance to share a network namespace and communicate over localhost. If you specify a container network without a name in a pod definition, it will be assigned to the default network.
+The Universal container runtime simplifies networking by allowing the containers of each pod instance to share a network namespace and communicate over localhost. If you specify a container network without a name in a pod definition, it will be assigned to the default network. If you have installed DC/OS [our AWS templates](/docs/1.9/installing/cloud/aws/), the default network is `dcos`.
```

Comment:
I do not like the language here..  `localhost` is not the right word to use.   they will communicate over a private network. or a vlan.